src/pip/_internal/commands/debug.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import absolute_import
+
+import logging
+import sys
+
+from pip._internal.cli import cmdoptions
+from pip._internal.cli.base_command import Command
+from pip._internal.cli.cmdoptions import make_target_python
+from pip._internal.cli.status_codes import SUCCESS
+from pip._internal.utils.logging import indent_log
+from pip._internal.utils.misc import get_pip_version
+from pip._internal.wheel import format_tag
+
+logger = logging.getLogger(__name__)
+
+
+def show_value(name, value):
+    logger.info('{}: {}'.format(name, value))
+
+
+def show_sys_implementation():
+    logger.info('sys.implementation')
+    implementation = sys.implementation
+    with indent_log():
+        show_value('name', implementation.name)
+
+
+def show_tags(options):
+    tag_limit = 10
+
+    target_python = make_target_python(options)
+    tags = target_python.get_tags()
+    # tags = get_supported()
+
+    # Display the target options that were explicitly provided.
+    formatted_target = target_python.format_given()
+    suffix = ''
+    if formatted_target:
+        suffix = ' (target: {})'.format(formatted_target)
+
+    msg = 'Compatible tags: {}{}'.format(len(tags), suffix)
+    logger.info(msg)
+
+    if options.verbose < 1 and len(tags) > tag_limit:
+        limited = True
+        tags = tags[:10]
```

Comment:
This should use `tag_limit`.

```suggestion
        tags = tags[:tag_limit]
```